### ISSUE TITLE
How to solve "NaN or Inf found in input tensor."

### ISSUE THREAD

**Issue by dongguanting** [NVIDIA/Megatron-LM#375]
When I pre-trained the GPT model, I found that there will be such a "NaN or Inf found in input tensor." print out during the training process. I suspect that the loss is not calculated successfully. How to fix this bug?
![image](https://github.com/NVIDIA/Megatron-LM/assets/60767110/4ce0f928-ea96-42c2-a1e2-b24f74bfe971)

**Comment by dongguanting** [comment]
I solved it , because i used unmatched vocab.json and merge.bpe
